tools/docker/firecracker-runc-config.json

```diff
@@ -0,0 +1,164 @@
+{
```

Comment:
The difference lies in `--privilege` being passed into docker granting `CAP_SYS_ADMIN` and various other capabilities. Meaning the config file will need far less capabilities than the example

Reply:
So what's the difference between the files?

Reply:
There isn't a difference due to needing to run as sudo, sorry if that wasn't clear. Did we want to remove the example one?

Reply:
The example one is the one customers will see and edit for try out firecracker-containerd. This one is the one customers don't have to care, but we'd like to use that inside integration tests (inside Docker).

So, I'd like to keep both of them.

Reply:
If they're identical, I'd prefer that our integration tests copy the example file rather than keeping both of them in source control.  It's hard to ensure that the files will remain identical otherwise.

(If they're intentionally different, which is fine, we should be clear about what those differences are.)